ui/src/pages/Admin/Edit/UploadStory.tsx

```diff
@@ -748,8 +777,15 @@ export const UploadStory: React.FC<StoryProps> = ({
                   id="tags-outlined"
                   name="tags"
                   freeSolo
-                  options={tagOptions ? tagOptions : [""]}
+                  forcePopupIcon
+                  options={tagOptions || [""]}
```

Comment:
Why default to `[""]` vs. an empty array `[]`?

Reply:
no real reason... i'll changed it to `[]` so it looks cleaner